src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1260,6 +1260,24 @@
   <data name="DoNotUseStackallocInLoopsMessage" xml:space="preserve">
     <value>Potential stack overflow. Move the stackalloc out of the loop.</value>
   </data>
+  <data name="PreferStreamReadAsyncMemoryOverloadsTitle" xml:space="preserve">
+    <value>Prefer Memory&lt;Byte&gt; overload for ReadAsync.</value>
+  </data>
+  <data name="PreferStreamReadAsyncMemoryOverloadsDescription" xml:space="preserve">
+    <value>All Stream based classes provide a ReadAsync overload that takes a Memory&lt;Byte&gt; as first argument. When possible, prefer the overload that takes a Memory&lt;Byte&gt;.</value>
```

Comment:
I feel the description can be more helpful and explain _why_ using the suggested overload is better.